tools/qemu_args.sh

```diff
@@ -50,8 +50,9 @@ if [ "$1" = "iommu" ]; then
     IOMMU_DEV_EXTRA=",iommu_platform=on,ats=on"
     IOMMU_EXTRA_ARGS="\
         -device intel-iommu,intremap=on,device-iotlb=on \
-        -device ioh3420,id=pcie.0,chassis=1\
+        -device ioh3420,id=pcie.0,chassis=1 \
```

Comment:
So the IOMMU enabling problem is caused by the lack of whitespace at the end?

Reply:
NoNo, it's the lack of OVMF, elaberated in https://github.com/asterinas/asterinas/issues/1517:

> I was wondering if we must use BIOS to enable IOMMU, why EFI can't?? After some search and chatgpt, it turns out we must use OVMF firmware to boot, and currently we don't configure OVMF upon enabling IOMMU......
>
> So the approach is simple, whether we add `-bios /usr/share/OVMF/OVMF_CODE.fd` in `tools/qemu_args.sh`, or just modify the part in `OSDK.toml` to `qemu.args = "$(./tools/qemu_args.sh iommu -ovmf)"`, no need to worry about BIOS any more...

Reply:
Ok. It took you quite some time to find out that `-ovmf` is needed to enable IOMMU. Could you add a comment here to share you newly-gained knowledge?

```toml
# Explain why add -ovmf is important
qemu.args = "$(./tools/qemu_args.sh iommu -ovmf)"
```